_shared_content/automate/actions.md

```diff
@@ -18,11 +18,11 @@ The Actions Library lists all available actions in playbooks with their detailed
 
 | Name | Description |
 | --- | --- |
-| [Get Event Field Common Values](library/sekoia-io/#get-event-field-common-values.md) | Retrieve the most common values of an ECS field based on the time window |
-| [List Assets](library/sekoia-io/#list-assets.md) | Retrieve detailed information about assets based on a filter |
-| [Search Alerts](library/sekoia-io/#search-alerts.md) | Retrieve detailed information about alerts (such as the urgency, name of the rule, etc… except events) based on a filter. |
-| [Get Alert](library/sekoia-io/#get-alert.md) | Retrieve detailed alert information such as the urgency, name of the rule, pattern, etc… except events. |
-| [Get Events](library/sekoia-io/#get-events.md) | Retrieve events based on a search. This action is equivalent to a search on the event page and takes into consideration 3 parameters: a query with filters (`source.ip=xx.xxx.xx`), and earliest time/latest time: two dates to determine the date range of the search. |
+| [Get Event Field Common Values](https://docs.sekoia.io/xdr/features/automate/library/sekoia-io/#get-event-field-common-values) | Retrieve the most common values of an ECS field based on the time window |
```

Comment:
```suggestion
| [Get Event Field Common Values](../library/sekoia-io/#get-event-field-common-values) | Retrieve the most common values of an ECS field based on the time window |
```

Reply:
Not sure but I think it's better to put relative path like this. It was not working before because `../` was missing at the start of the relative path.